tensorflow/python/layers/normalization.py

```diff
@@ -567,6 +567,7 @@ def batch_normalization(inputs,
       fused=fused,
       trainable=trainable,
       name=name,
+      dtype=inputs.dtype.base_dtype,
```

Comment:
If you haven't tested this as you do for conv3d, I highly recommend you to split this into another pull request.

Reply:
Yes, if you are working on batch norm it's better to remove it from my pull request.

Reply:
My colleague is working on that. I saw you have removed this. Thank you!